### PR TITLE
zoom-us: 5.4.53350.1027 -> 5.4.57862.0110

### PR DESCRIPTION
**Pull request by Mic92**
Replacing zooms qt library has let to incompatibilities between
qt versions used in Nix and zoom. The new approach just sets
LD_LIBRARY_PATH. LD_LIBRARY_PATH is also used internally by zoom
for its own vendored library.

The package has been tested end-to-end in a meeting.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by lovesegfault** [comment]
Tested and working on sway. I can join a meeting and use all features I need.

**Comment by lovesegfault** [comment]
Result of `nixpkgs-review pr 109602` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>zoom-us</li>
  </ul>
</details>

**Comment by Mic92** [comment]
Do we need to backport this?

**Comment by Mic92** [comment]
https://github.com/NixOS/nixpkgs/pull/109612

File: pkgs/applications/networking/instant-messengers/zoom-us/default.nix [hunk 1]
```diff
@@ -1,128 +1,119 @@
...
+    xorg.libXrender
+    zlib
+    xorg.xcbutilimage
+    xorg.xcbutilkeysyms
+    xorg.libXfixes
+    xorg.libXtst
+  ] ++ lib.optional (pulseaudioSupport) libpulseaudio);
+
+in stdenv.mkDerivation {
+  name = "zoom-${version}";
```
Comment (hmenke) [inline pkgs/applications/networking/instant-messengers/zoom-us/default.nix:73]: Why did you change
```nix
{
  pname = "zoom-us";
  inherit version;
}
```
to
```nix
{
  name = "zoom-${version}";
}
```
Just curious.

Comment (Mic92) [reply pkgs/applications/networking/instant-messengers/zoom-us/default.nix:73]: This was a mistake. At some point I switched to fhsUserEnv where I had to strip the version number.
